rel-eng/completions/bash/openshift

```diff
@@ -0,0 +1,3146 @@
+    two_word_flags=()
+    flags_with_completion=()
+    flags_completion=()
+
+    flags+=("--help")
+    flags+=("-h")
+    flags+=("--no-headers")
+    flags+=("--output=")
+    two_word_flags+=("-o")
+    flags+=("--output-version=")
+    flags+=("--quiet")
+    flags+=("-q")
+    flags+=("--source=")
+    two_word_flags+=("-f")
+    flags+=("--template=")
+    two_word_flags+=("-t")
+
+    must_have_one_flag=()
+    must_have_one_noun=()
+}
+
+_openshift_ex_options()
+{
+    last_command="openshift_ex_options"
+    commands=()
+
+    flags=()
+    two_word_flags=()
+    flags_with_completion=()
+    flags_completion=()
+
+    flags+=("--help")
+    flags+=("-h")
+
+    must_have_one_flag=()
+    must_have_one_noun=()
+}
+
+_openshift_ex()
+{
+    last_command="openshift_ex"
+    commands=()
+    commands+=("tokens")
+    commands+=("ipfailover")
+    commands+=("router")
+    commands+=("registry")
+    commands+=("build-chain")
+    commands+=("bundle-secret")
+    commands+=("options")
+
+    flags=()
+    two_word_flags=()
+    flags_with_completion=()
+    flags_completion=()
+
+    flags+=("--allow_dynamic_housekeeping")
+    flags+=("--alsologtostderr")
+    flags+=("--api-version=")
+    flags+=("--auth-path=")
+    flags+=("--boot_id_file=")
+    flags+=("--certificate-authority=")
+    flags+=("--client-certificate=")
+    flags+=("--client-key=")
+    flags+=("--cluster=")
+    flags+=("--config=")
+    flags+=("--container_hints=")
+    flags+=("--context=")
+    flags+=("--docker=")
+    flags+=("--docker_root=")
+    flags+=("--docker_run=")
+    flags+=("--enable_load_reader")
+    flags+=("--global_housekeeping_interval=")
+    flags+=("--google-json-key=")
+    flags+=("--help")
+    flags+=("-h")
+    flags+=("--housekeeping_interval=")
+    flags+=("--httptest.serve=")
+    flags+=("--insecure-skip-tls-verify")
+    flags+=("--log_backtrace_at=")
+    flags+=("--log_cadvisor_usage")
+    flags+=("--log_dir=")
+    flags+=("--log_flush_frequency=")
+    flags+=("--logtostderr")
+    flags+=("--machine_id_file=")
+    flags+=("--match-server-version")
+    flags+=("--max_housekeeping_interval=")
+    flags+=("--namespace=")
+    two_word_flags+=("-n")
+    flags+=("--server=")
+    flags+=("--stderrthreshold=")
+    flags+=("--token=")
+    flags+=("--user=")
+    flags+=("--v=")
+    flags+=("--validate")
+    flags+=("--vmodule=")
+
+    must_have_one_flag=()
+    must_have_one_noun=()
+}
+
+_openshift_version()
+{
+    last_command="openshift_version"
+    commands=()
+
+    flags=()
+    two_word_flags=()
+    flags_with_completion=()
+    flags_completion=()
+
+    flags+=("--help")
+    flags+=("-h")
+
+    must_have_one_flag=()
+    must_have_one_noun=()
+}
+
+_openshift_infra_router_version()
+{
+    last_command="openshift_infra_router_version"
+    commands=()
+
+    flags=()
+    two_word_flags=()
+    flags_with_completion=()
+    flags_completion=()
+
+    flags+=("--help")
+    flags+=("-h")
+
+    must_ha
```

Comment:
Does this work?  Isn't the command name "osc"   ?  Where does this full "openshift" name come from?